clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoClient.java

```diff
@@ -44,6 +44,10 @@ public class GravitinoClient implements SupportsMetalakes, Closeable {
 
   private static final ObjectMapper MAPPER = JsonUtils.objectMapper();
 
+  public static final String API_METALAKES_ALTER_LOAD_OR_DROP_PATH = "api/metalakes/";
+
+  public static final String API_METALAKES_CREATE_AND_LIST_PATH = "api/metalakes";
```

Comment:
Can you please use a more concise name for these two variables, it seems too verbose currently.